packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -203,10 +205,12 @@ import 'basic_types.dart';
 @immutable
```

Comment:
just above the "See also" above, it would be good to continue the story showing what name it ends up being exposed as, and how you refer to it from TextStyle.